source/openapi3/split.yaml

```diff
@@ -1581,19 +1552,19 @@ tags:
   - name: Open Agreements
     description: >
       An Open Agreement is essentially an Agreement template with no specific
-      authoriser. Each time an Open Agreement is accepted by either a Split
-      account holder or anyone, the authoriser is added to your Contacts list
-      and a new Agreement is automatically created between the Open Agreement
-      initiator and the authoriser.
+      authoriser. Each time an Open Agreement is accepted by anyone, the 
+      authoriser is added to your Contacts list and a new Agreement is 
+      automatically created between the Open Agreement initiator and the authoriser.
 
 
       An Open Agreement can be accepted multiple times by different parties and
-      the result is the same: A new Agreement. Additionally, an Open Agreement
-      can be accepted by anybody, not just other Split users. This is achieved
-      by using our [Instant Account Verification
-      process](http://help.split.cash/bank-accounts/instant-account-verification-iav)
-      as part of accepting an [Open
-      Agreement](https://help.split.cash/agreements/open-agreement).
+      the result is always the same: A new Agreement. 
+      
+      
+      Additionally, an Open Agreement can be accepted by anybody. This is achieved
+      by using our [Instant Account Verification process]
+      (http://help.split.cash/bank-accounts/instant-account-verification-iav)
```

Comment:
Does this link look correct in the web page? Some Markdown parsers don't like links being split across two lines like this.